.npmignore

```diff
@@ -2,7 +2,7 @@ node_modules/
 .npm
 .git/
 vv/
-venv/
+venv*
```

Comment:
I use venv36 for testing on python 3 and venv for python 2, was doing a release the other time and noticed the package was over 130mb because it took the venv36 files, that package was canceled before upload.